Userland/Libraries/LibCore/EventLoop.h

```diff
@@ -1,6 +1,6 @@
 /*
  * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
- * Copyright (c) 2022, kleines Filmröllchen <malu.bertsch@gmail.com>
+ * Copyright (c) 2022, kleines Filmröllchen <filmroellchen@serenityos.org>
```

Comment:
Sanity check: Did you replace yourself, or did you accidentally kick out someone else from the copyright list?

Reply:
Replaced myself